Fix UI colors

XanderD99 opened the pull request:
### Thanks a lot for contributing!<br>

Hey, I just found some more UI inconsistencies and tried fixing them all at once.

I removed the "TalkerScreenTheme" class and changed that option with the LogColors property. So now by default all colors for text and background and such just use the theme set by the user. 

The LogCards still use the color that is found in the LogColors map. The border of the card uses the color as it is provided and the background is the log color with some opacity added to it. The makes it readable on any theme (at least the colors I tested).

I also changed the default state of the expanded logs to false, if you don't want this I can revert that. I changed that so that the main feed can contain more data and you can then only expand those you want to see more details about. To give errors some more info I set the message to the errorType if no message is given. Errors always show some minor information this way.

If users want to set some custom theme in the log screen they can still wrap it in a Theme widget. Perhaps if needed someone can make a "TalkerTheme" widget that can be used to wrap the talkerScreen?

Commit:
💄 Update UI

remove talkerscreen theme in favor of just the Theme
add utility function to LogColors to get color based on log data
use Theme.of(context) consistently

Commit:
add const where needed

ArinFaraj committed:
fix(riverpod_logger): duplicated state logs fixes #241

XanderD99 committed:
Merge branch 'master' into fix/flutter-ui-colors

Commit:
fix bloc data logs

Frezyx committed:
Release v4.3.3

Frezyx committed:
Release v4.3.4

Commit:
fix card timestamp

Frezyx committed:
Merge branch 'master' into pr/242

Frezyx commented:
Hello @XanderD99 !
I've already looked at how the updated log screen looks like

For my opinion it's looks very nice. But it's huge breaking change for develeopers who already used talker_flutter to create self logging packages or logs screen customization. 

And now I'm thinking about how we can support old TalkerScreenTheme with Deprecated annotation and new UI

XanderD99 commented:
I think the best way to have that backwards compability is to keep the TalkerScreenTheme (bring it back in this case) but indeed mark it depricated. 
Then when someone uses the TalkerScreenTheme in there project just wrap the whole TalkerScreen in a Theme that uses the colors set in TalkerScreenTheme.

That way the TalkerScreenTheme is removed from al the UI itself but is still available to the other developers until it is completely removed after some time.

Hirakota commented:
It would be batter to change typedef LogColors from Map<TalkerLogType, Color> to Map<String, Color> it's will allow to create custom log types with custom colors

XanderD99 committed:
Merge branch 'master' into fix/flutter-ui-colors

Commit:
add talkertheme back as depricated

XanderD99 commented:
hey @Frezyx I updated the code and added TalkerTheme back to TalkerScreen. I marked is as depricated however, but when it is set it uses logColors from the talkerTheme and it wraps the TalkerView with a theme that should set background and card colors accordingly. 

I haven't fully tested this yet so this should be verified.
